thrall/app/lib/ElasticSearch.scala

```diff
@@ -131,19 +131,25 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
       .executeAndLog(s"removing exports from image $id")
       .incrementOnFailure(failedExportsUpdates) { case e: VersionConflictEngineException => true }
 
-  def applyImageMetadataOverride(id: String, metadata: JsValue)(implicit ex: ExecutionContext): Future[UpdateResponse] =
+  def applyImageMetadataOverride(id: String, metadata: JsValue)(implicit ex: ExecutionContext): Future[UpdateResponse] = {
+    val userMetaData = metadata \ "data"
+    val lastModified = metadata \ "lastModified"
     prepareImageUpdate(id)
       .setScriptParams(Map(
-        "userMetadata" -> asGroovy(metadata),
-        "lastModified" -> asGroovy(JsString(currentIsoDateString))
+        "userMetadata" -> asGroovy(userMetaData),
+        "lastModified" -> asGroovy(lastModified)
       ).asJava)
       .setScript(
-        "ctx._source.userMetadata = userMetadata;" +
-          refreshEditsScript +
-          updateLastModifiedScript,
+      s"""| if (ctx._source.lastModified && ctx._source.lastModified > lastModified) {} else {
```

Comment:
Would we want them to be the same? Technically the metadata was updated at some stage, and now we're updating the image. This time difference could be useful?